### DISCUSSION TITLE
GLIBC version mismatch

### DISCUSSION THREAD

**Discussion by bhaswa** [microsoft/onnxruntime#20403] [Other Q&A] [answered]
Hi,

I am trying to run the the library on an edge device, getting the below issue. My host machine GLIBC version is 2.23

/lib/libc.so.6: version `GLIBC_2.28' not found (required by /usr/lib/libonnxruntime.so.1.17.3)
/lib/libm.so.6: version `GLIBC_2.27' not found (required by /usr/lib/libonnxruntime.so.1.17.3)
/lib/libm.so.6: version `GLIBC_2.29' not found (required by /usr/lib/libonnxruntime.so.1.17.3)

Can you suggest to resolve the issue ?

**Comment by snadampal** [comment, marked as answer]
GLIBC_2.28 requirement is coming from manylinux2_28 base image used for building the onnxruntime wheels. I think you can try building onnxruntime from sources on your machine with older GLIBC to workaround this.
